src/components/RestaurantDetails/components/Offers/Offers.js

```diff
@@ -0,0 +1,55 @@
+//Libraries
+import React from 'react';
+import PropTypes from 'prop-types';
+
+//Helpers
+import offerReader from './readers/offerReader';
+
+//Icons
+import offerIcon from '../../../../images/offer.svg';
+
+//CSS
+import './Offers.css';
+
+function offerTemplate(offer) {
+  const discount = offerReader.discount(offer);
+  const coupon = offerReader.coupon(offer);
+  return (
+    <div className='offer' key={coupon}>
+      <div>
+        <img src={offerIcon} alt='offer' />
+      </div>
+      <span>
+        {discount} | Use code {coupon}
+      </span>
+    </div>
+  );
+}
+
+function renderOffers(offers) {
+  return offers.map(offerTemplate);
+}
+function Offers(props) {
+  const { offers } = props;
+  return (
+    <div className='offers'>
+      <div className='heading'>offers</div>
```

Comment:
This heading will be visible even if you don't have any offers, make it conditional